CHANGES.rst

```diff
@@ -7,7 +7,7 @@ Changelog
 - Add config option to select cipher for ssh connection (dasm)
 - Fix deprecation warnings for pytest (dasm)
 - Support pytest-xdist >= 1.26.0 (dasm)
-- Support pytest >= 3.6.1 (version supported by pytest-xdist without a bug) (dasm)
+- Support pytest >= 4.0.0 (dasm)
```

Comment:
sorry, I meant to raise the version of pytest-cloud itself, this change was good

Reply:
Oh, makes sense.